cubed/array_api/creation_functions.py

```diff
@@ -207,8 +215,8 @@ def _linspace(x, *arrays, size, start, step, endpoint, linspace_dtype, block_id=
     )
 
 
-def meshgrid(*arrays, indexing="xy"):
-    if len({a.dtype for a in arrays}) > 1:
+def meshgrid(*arrs, indexing="xy") -> Tuple["Array", ...]:
```

Comment:
Can this be kept as `arrays`? I'm trying to follow the naming in the array API spec: https://data-apis.org/array-api/latest/API_specification/generated/signatures.creation_functions.meshgrid.html